### ISSUE TITLE
Is boxbox dead?

### ISSUE THREAD

**Issue by gamecubate** [incompl/boxbox#39]
No significant updates in 3 years. I get the feeling that you got tired of it. For a project of this size, I can understand that. But if so, that's too bad. boxbox was simple, well documented and therefore a pleasure to work with.

**Comment by incompl** [comment]
I still intend to maintain it and respond to issues and bugs but I don't plan on adding new features. I really only add features when I'm actively using something because that lets me understand what is a good idea or bad idea in practice. I'm not doing any game development at the moment, so that's really what it comes down to. If somebody else wanted to make a feature roadmap for boxbox and add new stuff I'd be totally down to talk about it!

**Comment by gamecubate** [comment]
In hiatus, then. I understand perfectly. No sense adding features for features' sake. That said, I did need one (feature) -- the ability to destroy joints. Looking at your code I see some refactoring would be needed. I'm not sure it's worth the effort given I'd probably need composites and fixtures as well, as opposed to just using box2dweb. I did see a fork of your projects at http://topheman.github.io/boxbox/ but it doesn't add the functionality I need.

I may decide to clone boxbox and branch out of master to create a Joint object and give it a destroy function property, among others. If I do that, I may send a pull request your way, if that helps.
